### ISSUE TITLE
View and chat buttons in live conversations are missing accessible names

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/spreed#7531]

**Comment by nickvergessen** [comment]
I assume this is about the top bar?

**Comment by AndyScherzinger** [comment]
Also in the top bar but also in other areas.

**Comment by nickvergessen** [comment]
Okay checked the screenshots after I got access,
it's about all the buttons in the top bar while being in a call. But it's the old topbar which had the grid view icon there and not the media control.
So we should be extra careful and check all buttons of the new topbar
